src/app/dialog/attribute-type/form/constraint-config/coordinates/coordinates-constraint-config-form.component.html

```diff
@@ -24,35 +24,58 @@
   <div class="form-group">
     <label [for]="controls.Precision"
            i18n="@@constraint.coordinates.precision.label">
-      Fractional digits
+      Precision
     </label>
-    <input [id]="controls.Precision"
-           [formControlName]="controls.Precision"
-           [class.is-invalid]="precisionControl.touched && precisionControl.errors"
-           class="form-control"
-           min="0"
-           max="10"
-           type="number">
-    <div *ngIf="precisionControl.touched && precisionControl.errors?.required"
-         class="invalid-feedback"
-         i18n="@@constraint.coordinates.precision.error.integer">
-      You need to enter a number of fractional digits.
-    </div>
-    <div *ngIf="precisionControl.touched && precisionControl.errors?.pattern"
-         class="invalid-feedback"
-         i18n="@@constraint.coordinates.precision.error.integer">
-      You need to enter a valid integer number.
-    </div>
-    <div *ngIf="precisionControl.touched && precisionControl.errors?.min"
-         class="invalid-feedback"
-         i18n="@@constraint.coordinates.precision.error.min">
-      You need to enter a number greater than or equal to {{precisionControl.errors?.min?.min}}.
-    </div>
-    <div *ngIf="precisionControl.touched && precisionControl.errors?.max"
-         class="invalid-feedback"
-         i18n="@@constraint.coordinates.precision.error.max">
-      You need to enter a number lesser than or equal to {{precisionControl.errors?.max?.max}}.
-    </div>
+    <select [id]="controls.Precision"
+            [formControlName]="controls.Precision"
+            class="custom-select"
+            data-test="coordinates-constraint-precision-select">
+      <ng-container *ngIf="formatControl.value === coordinatesFormat.DecimalDegrees; else dmsPrecisions">
+        <option i18n="@@constraint.coordinates.precision.dd.0"
+                value="0">
+          0 digits (~ 100 km)
```

Comment:
Wouldn't it be better to create numeric enums or constants and iterate over it? Also translation will be quite easier.

Reply:
Thanks. I have changed that.